frontend/src/theme/components/Tag.ts

```diff
@@ -0,0 +1,38 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export type TagVariants = 'solid' | 'light'
+
+export const Tag: ComponentMultiStyleConfig = {
+  parts: [],
+  baseStyle: {
+    label: {
+      textStyle: 'caption-1',
+    },
+  },
+  variants: {
+    solid: {
+      container: {
+        textColor: 'secondary.700',
+        bgColor: 'success.400',
+      },
+    },
+    light: {
+      container: {
+        bgColor: 'primary.100',
+        textColor: 'primary.400',
+      },
+    },
+  },
+  sizes: {
+    uni: {
```

Comment:
leave as default `md` key is fine, don't need add new key; this keeps the application sizes consistent

Reply:
thanks for input, changed!